packages/block-library/src/template-part/index.php

```diff
@@ -63,7 +63,7 @@ function render_block_core_template_part( $attributes ) {
 	}
 	$content = do_shortcode( $content );
 
-	return str_replace( ']]>', ']]&gt;', $content );
+	return '<div class="wp-block-template-part">' . str_replace( ']]>', ']]&gt;', $content ) . '</div>';
```

Comment:
Do you think this going to have unexpected consequences?

Reply:
Im not sure, but I don't think so and it doesn't seem to so far.  Why should it just 'dump' all the inner content without a wrapper?  And as noted above, we will need a wrapper to apply it as a semantic element in the future as well.

Reply:
Cool! that makes sense to me 👍